src/Repositories/User/UserModel.ts

```diff
@@ -0,0 +1,16 @@
+import * as mongoose from 'mongoose';
+
+import UserSchema from './UserSchema';
+import IUserModel from './IUserModel';
+
+export const userSchema = new UserSchema({
+     collection: 'user',
+});
+
+export const userModel: mongoose.Model<IUserModel> = mongoose.model<IUserModel>
+(
+     'User',
+     userSchema,
+     'User',
+     true,
```

Comment:
can you please recheck this

Reply:
**(line 10-15)**
- Defining a model using
![pic14312](https://user-images.githubusercontent.com/72728297/98335300-b60db900-202a-11eb-8607-5ceacd239ba6.png)
 function mongoose.model() which has 4 arguments [name='User', schema='userSchema', collections='User',initKey='false' ]